portfolio/src/main/webapp/script.js

```diff
@@ -29,6 +29,12 @@ function addRandomGreeting() {
 
 function fetchMessage() {
     fetch("/data").then(request => request.text()).then((textResponse) => {
-        document.getElementById("message-container").innerText = textResponse;
+        var text = textResponse.substring(1, textResponse.length-2).split(',')
```

Comment:
We can use request.json() instead of request.text() and directly access the JSON array rather than parsing the text into an array manually ourselves. Let me know if you need help with this.